src/__tests__/spectral.test.ts

```diff
@@ -1,10 +1,61 @@
-import { DiagnosticSeverity } from '@stoplight/types';
+import { DiagnosticSeverity, Dictionary } from '@stoplight/types';
 import { isParsedResult, Spectral } from '../spectral';
 import { IParsedResult, RuleFunction } from '../types';
 
 const merge = require('lodash/merge');
 
+const oasRuleset = JSON.parse(JSON.stringify(require('../rulesets/oas/index.json')));
+const oas2Ruleset = JSON.parse(JSON.stringify(require('../rulesets/oas2/index.json')));
+const oas3Ruleset = JSON.parse(JSON.stringify(require('../rulesets/oas3/index.json')));
+
 describe('spectral', () => {
+  describe('loadRuleset', () => {
+    test('should support loading built-in rulesets', async () => {
+      const s = new Spectral();
+      await s.loadRuleset('spectral:oas2');
+
+      expect(s.rules).toEqual(
+        [...Object.entries(oasRuleset.rules), ...Object.entries(oas2Ruleset.rules)].reduce<Dictionary<unknown>>(
+          (oasRules, [name, rule]) => {
+            oasRules[name] = {
+              name,
+              ...rule,
+              formats: expect.arrayContaining([expect.any(String)]),
+              severity: expect.any(Number),
+              then: expect.any(Object),
+            };
+
+            return oasRules;
+          },
+          {},
+        ),
+      );
+    });
+
+    test('should support loading multiple built-in rulesets', async () => {
+      const s = new Spectral();
+      await s.loadRuleset('spectral:oas2', 'spectral:oas3');
```

Comment:
Can we get another test which is loading `spectral:oas` only, and show that both oas2 and oas3 rules are loaded? 

This is the final missing piece that will allow people to load one ruleset and get both, depending on which format is loaded. We probably should have got it done as part of #432 but it slipped by.

Reply:
It's done in https://github.com/stoplightio/spectral/pull/461